core/templates/dev/head/pages/exploration_player/AudioControlsDirective.js

```diff
@@ -35,12 +35,30 @@ oppia.directive('audioControls', [
           // distinguishing which audio directive is currently playing audio.
           var directiveId = Math.random().toString(36).substr(2, 10);
 
+          var getCurrentAudioTranslation = function() {
+            var currentAudioLanguageCode =
+              AudioTranslationManagerService.getCurrentAudioLanguageCode();
+            return $scope.getAudioTranslations()[currentAudioLanguageCode];
+          };
+
           $scope.AudioPlayerService = AudioPlayerService;
 
           $scope.IMAGE_URL_REWIND_AUDIO_BUTTON = (
             UrlInterpolationService.getStaticImageUrl(
               '/icons/rewind-five.svg'));
 
+          $scope.isAudioAvailableInCurrentLanguage = function() {
+            return Boolean(getCurrentAudioTranslation());
+          };
+
+          $scope.isCurrentAudioTranslationNeedingUpdate = function() {
+            return getCurrentAudioTranslation().needsUpdate;
+          };
+
+          $scope.audioNotAvailableMessage = 'Not available in ' + 
```

Comment:
Think this would work for i18n of the tooltip text? https://github.com/angular-translate/angular-translate/issues/163#issuecomment-23846414

If we want to do that, maybe maintain $scope.currentAudioLanguage instead of retrieving it each time, and update it on-change. If all this is not straightforward, though, feel free to punt on it (but please file an issue).